blocks/abp-review/utils/sendHelixData.js

```diff
@@ -21,13 +21,20 @@ const sendHelixData = ({
   if (visitorId) {
     data.push({ name: 'VisitorId', value: visitorId });
   }
+
   if (page) {
     data.push({ name: 'Page', value: page });
   }
+
   if (timeSpentInSeconds) {
     data.push({ name: 'TimeSpentInSeconds', value: timeSpentInSeconds });
   }
 
+  if (profileDetails) {
```

Comment:
Since `profileDetails` is an object, which will return `true` even if it's empty, you may want to check for `name` and `email` keys within it before pushing their values

<img width="231" alt="Screenshot 2023-05-15 at 8 52 51 PM" src="https://github.com/adobecom/abpdocs/assets/4350668/f12566c0-ff1f-4ce9-8c6c-0c94f58a74f5">